homeassistant/components/progettihwsw/strings.json

```diff
@@ -2,8 +2,10 @@
   "config": {
       "error": {
           "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
-          "unknown": "[%key:common::config_flow::error::unknown%]",
-          "wrong_info_relay_modes": "Relay mode selection must be monostable or bistable."
+          "unknown": "[%key:common::config_flow::error::unknown%]"
+      },
+      "abort": {
+          "already_setup": "[%key:common::config_flow::abort::single_instance_allowed%]"
```

Comment:
usually the convention is to all this `already_configured` then you can pick from these common keys:

https://github.com/home-assistant/core/blob/dev/homeassistant/strings.json#L51-L53